content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -133,8 +133,8 @@ If developers use additional functions like `WithValues` or `WithName` in
 their components, then log entries contain additional information that gets
 passed into functions by their caller.
 
-Currently this is gated behind the `StructuredLogging` feature gate and
-disabled by default. The infrastructure for this was added in 1.24 without
+Currently this is gated behind the `ContextualLogging` feature gate and
```

Comment:
```suggestion
For Kubernetes {{< skew currentVersion >}}, this is gated behind the `ContextualLogging`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and is
```

Reply:
Done.

I also updated the feature gate description.